### ISSUE TITLE
The system fails to predict bboxes.

### ISSUE THREAD

**Issue by ZichengDuan** [chenyuntc/Simplified-FasterRCNN#212]
@chenyuntc 

HI, Mr. Chen, I downloaded the latest version of your implementation, however, I encountered some vital issues when training the network. 

The software fails to generate predicted boxes, and the problem seems to raise from the `predict()` function in `faster_rcnn.py`. Specifically in:  
`prob = (F.softmax(at.totensor(roi_score), dim=1))`. By the way, the roi_score here also seems to be abnormal. 

the `probability (prob)` that is later sent into function `_suppress()` is too small, so that in function `_suppress()`, the boolean result from` keep = nms(cls_bbox_l, prob_l,self.nms_thresh)` is thus always `False`, since no predicted anchor has a score greater than 0.7 (i.e. ThreshHold). 

At first, I thought this results from the fact that the network is good enough to predict good boxes, however, I found that the Test mAP is unexpectedly low compares to your old version. The mAP for the first 4 epoch is 0.0017, 0.012, 0.0044, 0.0045.

Could you please give me some instructions on how to figure this problem? Thanks.
